lib/checkout_sdk/instruments/previous/instruments.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```

Comment:
I think is better just keep the parent for `require`, I mean put this references on

lib/checkout_sdk/instruments/instruments.rb

and probably just a marker like `## Previous`

So that in the future we should worry about one file per module.